tests/integration/test_integration.py

```diff
@@ -250,6 +253,9 @@ def test_single_trace_too_large(encoding, monkeypatch):
     monkeypatch.setenv("DD_TRACE_API_VERSION", encoding)
 
     t = Tracer()
+    t.configure(
+        partial_flush_enabled=False,
```

Comment:
Same here, we'll need 2 variants of this test one where we update the assertions with the default behavior, and then this one with partial flushing disabled.